open_spiel/games/pig.cc

```diff
@@ -234,10 +222,18 @@ std::vector<std::pair<Action, double>> PigState::ChanceOutcomes() const {
   SPIEL_CHECK_TRUE(IsChanceNode());
   std::vector<std::pair<Action, double>> outcomes;
 
-  // Chance outcomes are labelled 0+, corresponding to rolling 1+x.
-  outcomes.reserve(dice_outcomes_);
-  for (int i = 0; i < dice_outcomes_; i++) {
-    outcomes.push_back(std::make_pair(i, 1.0 / dice_outcomes_));
+  if (piglet_) {
+    // Chance outcomes are labelled 0 or 1, corresponding to rolling 1 or not 1 respectively
+    outcomes.reserve(2);
+    outcomes.push_back(std::make_pair(0, 1.0 / dice_outcomes_));
```

Comment:
Is it expected that dice_outcomes_ is used here, since the original piglet uses a fair coin?
If it is, maybe explain better how dice_outcomes_ in piglet mode? It might be clearer to introduce another parameter instead of changing the meaning of dice_outcomes_ depending on whether we are in piglet mode or not.

Reply:
Good point, this was unclear. It is indeed intended, but I added a paragraph to explain the Piglet variant in the header file. The meaning of dice_outcomes_ does not change, it still indicates the number of sides of the die that's used. Original Piglet uses a 2-sided die (a coin), original Pig uses a 6-sided die.